kvision-modules/kvision-maps/src/main/kotlin/externals/leaflet/control/Control.kt

```diff
@@ -0,0 +1,62 @@
+package externals.leaflet.control
+
+import externals.leaflet.PositionsUnion
+import externals.leaflet.control.Control.ControlOptions
+import externals.leaflet.control.Control.LayersObject
+import externals.leaflet.core.Class
+import externals.leaflet.layer.Layer
+import externals.leaflet.map.LeafletMap
+import org.w3c.dom.HTMLElement
+
+/**
+ * `Control` is a base class for implementing map controls. Handles positioning. All other controls
+ * extend from this class.
+ *
+ * See [`https://leafletjs.com/reference.html#control`](https://leafletjs.com/reference.html#control)
+ */
+@JsModule("leaflet")
+@JsNonModule
+abstract external class Control<T : ControlOptions>(
+    options: T = definedExternally
+) : Class {
+
+    open var options: T // ControlOptions
```

Comment:
I think I'll take out the generic option fields. I put them in because subclasses can't override them. But that's not much of an issue - I don't think users would be accessing this options field directly.